drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -357,26 +357,20 @@ function DragAndDropEditBlock(runtime, element, params) {
                             _fn.build.form.zone.renderZonesPreview();
                         },
                     },
-                    createDropdown: function(selectedUID) {
-                        var template = _fn.tpl.zoneDropdown;
-                        var dropdown = [];
+                    createCheckboxes: function(selectedZones) {
+                        var template = _fn.tpl.zoneCheckbox;
+                        var checkboxes = [];
                         var zoneObjects = _fn.build.form.zone.zoneObjects;
 
                         zoneObjects.forEach(function(zoneObj) {
-                            dropdown.push(template({
-                                uid: zoneObj.uid,
+                            checkboxes.push(template({
+                                zoneUid: zoneObj.uid,
                                 title: zoneObj.title,
-                                selected: (zoneObj.uid == selectedUID) ? 'selected' : '',
+                                checked: $.inArray(zoneObj.uid, selectedZones) != -1 ? 'checked' : '',
```

Comment:
@haikuginger The previous version of the code wasn't doing it, but we generally want to use strict comparison (`===` and `!==`).